pkg/controller/common/stackmon/config.go

```diff
@@ -159,8 +159,8 @@ type inputConfigData struct {
 	URL      string
 	Username string
 	Password string
-	IsSSL    bool
-	SSLPath  string
+	IsCA     bool
```

Comment:
I know `Is*` is the go convention but would `HasCA` not be better fitting here?

Reply:
++, I was tempted but I didn't dare.